bsip-0048.md

```diff
@@ -11,44 +11,40 @@
 # Abstract
 
 The possibility of creating assets on the BitShares Blockchain is a core feature
-of our platform. In the case of an ico, the ico-holders can create an asset with
+of our platform. In the case of an ICO, the ICO holders can create an asset with
 a fixed maximum supply. However this maximum supply can be changed by the
-asset creator (issuer) anytime. This exposes participators ico to the risk,
-of loosing value of their bought asset.
+asset creator (issuer) anytime. This exposes participants in the ICO to the risk
+of losing value of their purchased asset.
 
 With this BSIP a flag will be introduced which forbids the modification of the maximum
 supply of an issuer created asset.
 
 # Motivation
 
-The implementation of a flag, to disable the modification of the maximum supply for 
-issuer created asset, has effects for both sides. On the one side the security for the
-participators or non-issuer users increases, because the value of their assets (tokens)
-can't be manipulated by exposing the market to more created tokens. And on the other side
-it is good marketing for the ico-hoders, because they can advertise with their fixed
-maximum supply.
+The implementation of a flag to disable the modification of the maximum supply for 
+issuer created asset, has benefits bor both the participants and the ICO holder. Participants int the ICO are assured the value of their assets (tokens) will not be diluted by issuers creating more tokens. ICO holders are able to market the fact that the maximum supply is fixed.
 
 # Rational
 
-The `disable_modify_max_supply` is a new created flag in the `asset_issuer_permissions`.
+The `allow_modify_max_supply` is a newly created flag in the `asset_issuer_permissions`.
```

Comment:
We need to use the `disable_*` methodology here to be compatible with existing data and client applications. See https://github.com/bitshares/bitshares-core/pull/1375#pullrequestreview-164580457.

Reply:
Thank you for the reference. It is unfortunate, but necessary. I will change it back.